geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/CompositeConverter.scala

```diff
@@ -101,6 +100,8 @@ class CompositeConverter[I](val targetSFT: SimpleFeatureType, converters: Seq[(P
   override def processSingleInput(i: I, ec: EvaluationContext): Iterator[SimpleFeature] = ???
 
   override def process(is: InputStream, ec: EvaluationContext): Iterator[SimpleFeature] = ???
+
+  override def close(): Unit = indexedConverters.foreach(_.close())
```

Comment:
if one converter throws an exception closing will the others close properly? Question I guess is really around resource leaking.

Reply:
yeah, i was lazy b/c this was the old deprecated one, but I'll fix it

Reply:
fixed